src/Connection.php

```diff
@@ -97,4 +97,27 @@ public function getSchema(): SchemaInterface
 
         return $this->schema;
     }
+
+    /**
+     * Initializes the DB connection and sets default date and time format.
+     *
+     * This method is invoked right after the DB connection is established.
+     *
+     * The default implementation turns on `PDO::ATTR_EMULATE_PREPARES`, if {@see getEmulatePrepare()} is `true`.
+     */
+    protected function initConnection(): void
+    {
+        parent::initConnection();
+
+        $this->pdo->exec(
+            <<<SQL
```

Comment:
What benefits in changes settings of db? What about cases with default settings? 
This changes can be in documentation only. It's change user experience.

Reply:
When getting datetime values from Oracle they are returned in a specific format that is not suitable for proper parsing by `DateTime`.
Yes, sure it will need to be added to the documentation